src/requests/process-method.ts

```diff
@@ -71,8 +71,17 @@ function getParamSeparation(paramGroups: Dictionary<Parameter[]>): string[] {
     let def: string;
 
     if (groupName === 'query') {
-      def = 'const queryParams = new HttpParams();\n';
-      def += _.map(group, p => `queryParams.append('${p.name}', JSON.stringify(params.${p.name}));`).join('\n');
+      def = 'const queryParams = new HttpParams()\n';
```

Comment:
the HttpParams creation did not work correctly before

Reply:
Great. I am really disappointed by how HttpParams work comparing to original Http service.